tests/stub.c

```diff
@@ -33,7 +32,6 @@ _cmocka_run_group_tests_mpi(const char * name, const struct CMUnitTest tests[],
     allocator_init(A_MAIN, "MAIN", 256 * 1024 * 1024, 1, NULL);
     allocator_init(A_TEMP, "TEMP", 8 * 1024 * 1024, 1, A_MAIN);
 
-    message(0, "GADGET_VERSION : %s\n", GADGET_VERSION);
```

Comment:
Why avoid rebuilding stub.c on version changes? Too many relinks taking a long time?

Reply:
It means that any commit will rebuild all the tests, not just relink.